gradle/javadoc.gradle

```diff
@@ -14,7 +14,7 @@ task mockitoJavadoc(type: Javadoc) {
     title = "Mockito ${project.version} API"
     def javadocJdk6Hack = "jdk6-project-version-insert.min.js"
 
-    options.doclet "org.mockito.javadoc.JavadocExclude"
+    options.doclet "org.mockito.javadoc.JavadocExclude1_${JavaVersion.current().toString()[-1]}"
```

Comment:
It is a one-liner yet I find the name ugly ;)
Also this won't work with Java 10.